src/index.js

```diff
@@ -0,0 +1,77 @@
+import './index.css';
+// import _ from 'lodash';
+import Element from '../modules/element.js';
+import { removeAllCompleted, updateTask } from '../modules/status.js';
+import Task from '../modules/task.js';
+
+const taskObject1 = new Task();
+const elementObject1 = new Element();
+elementObject1.btnAddNew.addEventListener('click', () => {
+  if (elementObject1.titleInput.value) {
+    taskObject1.addTask();
+  }
+});
+taskObject1.retrieveData();
+window.addEventListener('load', () => {
+  document.querySelectorAll('.checkbox').forEach((element) => {
```

Comment:
-  Could you please initiate all variables at the beginning of the various scopes in order to keep JavaScript best practices